src/content/docs/dropins/payment-services/index.mdx

```diff
@@ -0,0 +1,23 @@
+---
+title: Payment Services overview
+description: Learn about the features of the Payment Services drop-in component.
+---
+
+import Link from '@components/Link.astro';
+import List from '@components/List.astro';
+
+The Payment Services drop-in renders the credit card form, allowing shoppers to efficiently enter their payment details. The drop-in currently supports credit/debit cards only.
+
+## Available Payment Services
+
+The Payment Services drop-in component consists of currently one container: `CreditCard`:
+
+- **Credit card container**: The `CreditCard` container is designed to allow placing an order with the available credit card details during the checkout process.
```

Comment:
Let's add a link to `dropins/payment-services/containers/credit-card`. I propose linking from `CreditCard`, just like you proposed in [containers.mdx](https://commerce-docs.github.io/microsite-commerce-storefront/dropins/payment-services/containers/) (which as far as I can tell cannot be reached).

```suggestion
- **Credit card container**: The [`CreditCard`](/dropins/payment-services/containers/credit-card) container is designed to allow placing an order with the available credit card details during the checkout process.
```